packages/icons/web/v2/forward_24.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="25" height="24" viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path id="Vector" d="M12.4999 14.9999V19.2328C12.4999 20.1019 13.5323 20.5573 14.1742 19.9713L23.1911 11.7384C23.6254 11.3419 23.6254 10.658 23.1911 10.2615L14.1742 2.02862C13.5323 1.44257 12.4999 1.89794 12.4999 2.76711V6.99994C2.79419 6.99997 1.49932 15.6512 1.61812 20.9583C1.63572 21.7445 2.91811 21.857 3.25823 21.148C4.60955 18.3309 7.30563 15 12.4999 14.9999Z" fill="#0F131A"/>
```

Comment:
The SVG should not have any `id` attributes.

Monochrome SVGs must use `currentColor` as their fill/stroke colors.